front/app/modules/commercial/machine_translations/citizen/components/PostShowTranslatedCommentBody.tsx

```diff
@@ -17,23 +18,20 @@ interface Props {
 const PostShowTranslatedCommentBody = ({
   translateButtonClicked,
   commentContent,
-  locale,
   commentId,
 }: Props) => {
-  const isMachineTranslationsEnabled = useFeatureFlag({
-    name: 'machine_translations',
-  });
-  const { data: translation } = useMachineTranslationByCommentId({
-    commentId,
-    machine_translation: {
-      locale_to: locale,
-      attribute_name: 'body_multiloc',
-    },
-    enabled: isMachineTranslationsEnabled,
```

Comment:
Just replaced the `useMachineTranslationByCommentId` with the general `useTranslation` hook, which is what we do elsewhere.